.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "Meshery-linkerd",
+	"image": "aisuko/golang:v1.14",
+	"runArgs": [ "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined" ],
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	"forwardPorts": [10001],
+
+	// Use 'settings' to set *default* container specific settings.json values on container create. 
+	// You can edit these settings after create using File > Preferences > Settings > Remote.
+	"settings": { 
+		"terminal.integrated.shell.linux": "/bin/bash",
+		"go.gopath": "/go",
+		"go.inferGopath": true,
+		"go.useLanguageServer": true
+	},
+
+	// Add the IDs of extensions you want installed when the container is created in the array below.
+	"extensions": [
+		"golang.Go",
+		"eamodio.gitlens"
+	],
+	
+	// Uncomment the next line to run commands after the container is created.
+	// "postCreateCommand": "go version",
+
+	// Comment out the next line to run as root
+	// "remoteUser": "vscode",
+	"remoteEnv": {
+		"GOPROXY":"https://goproxy.io,direct"
+	}
+}
```

Comment:
This devcontainer is pretty neat. I haven't seen this before.

Reply:
Yep, https://github.com/Aisuko/dev-container  we can debug the project into the container(It also support the remote docker daemon)